util/fscrypt_completion

```diff
@@ -0,0 +1,225 @@
+# bash completion for fscrypt                              -*- shell-script -*-
+
+# Output list of possible mount points
+_fscrypt_mountpoints()
+{
+    fscrypt status 2>/dev/null | \
+        command sed \
+            -e '/^\//!d' \
+            -e '/Yes$/!d' \
+            -e 's/ .*//'
+}
+
+# Output list of possible policy or protector IDs
+# $1: the mount point on which policies are looked for.
+# $2: the section (policy or protector) to retrieve
+_fscrypt_status_section()
+{
+    fscrypt status "$1" 2>/dev/null | \
+        command sed \
+            -e "/^${2^^} /",'/^$/!d' \
+            -e "s/${2^^}//" \
+            -e "s/\s.*//"
+}
+
+_fscrypt()
+{
+    local cur prev words cword split
+    local GLOBAL_OPTS="--verbose --quiet --help"
+    _init_completion -s -n : || return
```

Comment:
Can you add a brief comment about what `_init_completion()` does and what the `s` and `-n` options mean?  I can find it defined in `/usr/share/bash-completion/bash_completion`, but it was not at all obvious.

Reply:
`/usr/share/bash-completion/bash_completion` comes with the [scop/bash-completion](https://github.com/scop/bash-completion) toolbox.  That is a common way, in completion script context, to initialize completion variables thus.  Yes, I'll add a comment.